addons/l10n_it_edi_withholding/models/account_move.py

```diff
@@ -149,10 +150,10 @@ def _l10n_it_edi_get_extra_info(self, company, document_type, body_tree, incomin
             if withholding_tax:
                 withholding_taxes.append(withholding_tax)
             else:
-                message_to_log.append("%s<br/>%s" % (
+                message_to_log.append(Markup("<br/>").join((
```

Comment:
Sure, will change it back, but it this way in `l10n_it_edi` so I wanted to be consistent with these 3.
Should I also change the joins there? It's not much effort, but it could generate some fw-port conflict in the future.
Lemme know 😄